01_02_Combat_Rules.txt

```diff
@@ -999,3 +999,128 @@ retrieve a dropped weapon or stand back up if they have fallen.
+one entry point that they will use during the breach. If there is any ambiguity,
+they must indicate which room they will storm when the breach occurs. Which room
+you are breaching is usually obvious; it's the room on the other side of the 
+entry point from where your character is stacking up. On the rare occasion that
+one door leads to more than one room, a single room must be designated for 
+clearing. You cannot breach and clear a room that is not attached to the entry
+point that you stacked up on. 
+
+    If your turn starts and you are already stacked up, you may either leave the
+line of the stack up and take your turn normally, or skip your turn to remain in
+the stack up line. If you skip your turn in this manner, you will have the same
+number of actions during clearing as you would if you stacked up as the first
+act of your turn.
+
+    Any number of characters can stack up on a given entry point. The order of
+the characters in the lineup is important. Players can chose any position in the
+stackup order that hasn't yet been taken at the time that they declare their 
+character will stack up. So the first character to stack up could choose to get
+in the middle of the line, and the second character to use their turn to stack
+up could stack up in front if they so chose.  
+
+===== Breaching =====
+
+    Any time a character joins the stack up line, the character in the lead of
+the stack may call "breach!" At that time, regular initiative order is 
+interrupted, and the breach and clear phases occur during the turn of the 
+character who just got in line. After the clearing has resolved, play resumes
+after that character's turn.
+
+    Some weapons, tools or gadgets are commonly used during a breach maneuver.
+Examples include breaching charges, tear gas or flashbangs. If these are used,
+they take effect first in the breach. Keep in mind that a character using such
+a device will need a free hand to do so, and may need to spend one of their 
+clearing actions to re-draw their weapon. As such, devices are usually used by
+the characters in the 3rd or 4th positions.
+
+    Some rooms are defended with booby traps. In the case that a room or entry
+point has such perils, they take effect second. 
+
+    While devices are being resolved, all attackers and defenders roll a reflex
+save to determine the order of combat during the clearing phase. If the 
+defenders were unaware of the impending breach, then they roll reflex with a
+disadvantage and the attackers roll reflex with advantage.
+
+*Breaching Roll = Reflex Save + Weapon Reflex Mod + any mod from feats
+
+===== Clearing =====
+
+    Players take their turn in decending order of breaching roll. In other words,
+The character with the highest net reflex roll goes first, the character with 
+the lowest reflex goes last. Characters on the attacking team (the team that 
+stacked up) must use an Action to move from their position in the stack line
```

Comment:
It's been awkward during room clears when you're fourth in line but roll higher in the ordering than the player in first. Do they jump the line or wait and squander that glorious reflex roll? 
It's probably been as bad when the defenders roll higher than the attackers and can't shoot anything because nobody's entered the room yet.

Would it be feasible to stick an Entry phase between Breach and Clear, where the attackers all execute a movement action in stackup order to enter the room? I think it would reduce the purely mechanical obnoxiousness that linear action order imposes and also make these events even more bloodily (viscerally?) quick, which I think would feel authentic and novel.

Reply:
In the defender-high position, they simply hold actions and blast the door or try to shoot through walls. The way I have it written, you have the option of jumping line and moving through your entry point in reflex order, as long as someone isn't already blocking it. My hesitation to add an Entry phase is that I think some breaches don't necessarily involve physical entry of the room until the shooting has stopped. It may be that waste of reflex is a mechanical penalty for trying to squeeze your dudes through a small hole. I'll do a little more research and come back to this. I'm still considering adding the Entry Phase as you described it.